src/is_scene.py

```
@@ -46,102 +80,125 @@ async def is_scene(video, meta, imdb=None, lower=False):
+                                                    meta['nfo'] = True
+                                                    meta['auto_nfo'] = True
+                                                console.print(f"[green]NFO downloaded to {nfo_file_path}")
+                                        else:
+                                            meta['nfo'] = True
+                                            meta['auto_nfo'] = True
+                                    except Exception as e:
+                                        console.print("[yellow]Failed to download NFO file:", e)
+
+                        return release_name
+
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)